src/components/charts/svg-d3-line-chart.tsx

```diff
@@ -107,10 +108,13 @@ export const SvgD3LineChart = (props: LineChartProps) => {
         .text((d) => "Run " + d.group)
         .style("fill", (d) => myColor("run" + d.group) as string)
         .style("text-decoration", "underline")
+        .style("cursor", () => running ? "default" : "pointer")
         .style("font-size", 15)
       .on("click", (d) => {
+        if (!running){
```

Comment:
minor nitpick here: It might be more legible to me if the body of the if statement was indented.